specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-10-01-preview/containerInstance.json

```diff
@@ -0,0 +1,972 @@
+        },
+        "summary": "Get a list of container groups in the specified subscription and resource group.",
+        "description": "Get a list of container groups in a specified subscription and resource group. This operation returns properties of each container group including containers, image registry credentials, restart policy, IP address type, OS type, state, and volumes.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group that contains the container group.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContainerGroupListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerInstance/containerGroups/{containerGroupName}": {
+      "get": {
+        "operationId": "ContainerGroups_Get",
+        "x-ms-examples": {
+          "ContainerGroupsGet_Succeeded": {
+            "$ref": "./examples/ContainerGroupsGet_Succeeded.json"
+          },
+          "ContainerGroupsGet_Failed": {
+            "$ref": "./examples/ContainerGroupsGet_Failed.json"
+          }
+        },
+        "summary": "Get the properties of the specified container group.",
+        "description": "Gets the properties of the specified container group in the specified subscription and resource group. The operation returns the properties of each container group including containers, image registry credentials, restart policy, IP address type, OS type, state, and volumes.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group that contains the container group.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "containerGroupName",
+            "in": "path",
+            "description": "The name of the container group.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContainerGroup"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "ContainerGroups_CreateOrUpdate",
```

Comment:
>ContainerGroups_CreateOrUpdate [](start = 24, length = 30)

Is this a long running operation?

Reply:
No. We don't support long running at the moment. We are aware of the issue and have work item to fix that later.